docs/it/authorization_endpoint.rst

```diff
@@ -276,3 +276,73 @@ Esempio di Authorization Response dell'OP:
     http://rp-test.it/oidc/rp/callback/?code=a032faf23d986353019ff8eda96cadce2ea1c368f04bf4c5e1759d559dda1c08056c7c4d4e8058cb002a0c8fa9a920272350aa102548523a8aff4ccdb44cb3fa&state=2Ujz3tbBHWQEL4XPFSJ5ANSjkhd7IlfC&iss=http%3A%2F%2Fop-test%2Foidc%2Fop%2F
 
 
+Gestione degli errori
++++++++++++++++++++++
+
+In caso di errore, l'OP o il RP rappresentano i messaggi di anomalia relativi agli scambi OpenID
+Connect, come descritti nelle relative tabelle definite dalle `Linee Guida UX SPID`_.
+
+
+.. list-table:: 
+   :widths: 20 60 20
+   :header-rows: 1
+
+   * - **Claim**
+     - **Descrizione**
+     - **Supportato da**
+   * - **error**
```

Comment:
Errore

Reply:
Fixed in https://github.com/italia/spid-cie-oidc-docs/pull/77/commits/19a3b3ea839f2763010e98c7706453b7b38d4036